README.md

```diff
@@ -74,6 +74,18 @@ import { supportedLocales } from '@teclone/industries';
 console.log(supportedLocales);
 ```
 
+### Export types
+
+```javascript
+import { type Industry, industries } from '@teclone/industries/src/types'
```

Comment:
developers are not allowed to import from the src directory, the src directory contains the src code, which is written in typescript.

secondly, industries is not exported by industries is not exported by @teclone/industries/src/types.

Reply:
we can export Industry type in src/index.ts, to make it easier to import from the entry build file

Reply:
> developers are not allowed to import from the src directory, the src directory contains the src code, which is written in typescript.
> 
> secondly, industries is not exported by industries is not exported by @teclone/industries/src/types.

I did it, using NextJS with Typescript. But yes, it is better to export Industry Type from index.

Reply:
yea, it works actually as long as you allow allow node_modules to be processed by webpack/babel, but that is not encouraging, it will take a lot of time to build and transpile node_module packages.